packages/integration-sdk-runtime/src/execution/config.ts

```diff
@@ -70,6 +70,13 @@ function convertEnvironmentVariableValueForField(
       }
       break;
     }
+    case 'string[]': {
+      convertedValue = environmentVariableValue
+        ?.split(',')
+        .map((v) => v.trim())
+        .filter((v) => v.length > 0) || []
```

Comment:
I drew inspiration from the exact logic we use in the frontend: https://github.com/JupiterOne/web-integrations/blob/main/src/pages/InstanceManagement/Template/IntegrationInstanceForm/useIntegrationInstanceFormLogic.ts#L69